IBAnimatable/ContainerTransition.swift

```diff
@@ -5,22 +5,22 @@
 
 import UIKit
 
-public class ContainerTransition: NSObject {
+open class ContainerTransition: NSObject {
```

Comment:
I'm not sure that one should be `open` right now, as well as its properties and methods.